apps/web/github.talon

```diff
@@ -7,51 +7,87 @@ win.title: /github.com/
 focus search: key(s)
 go to notifications: insert("gn")
 go to dashboard: insert("gd")
-show keyboard shortcuts: key(?)
+keyboard shortcuts show: key(?)
+selection move down: key(j)
+selection move up: key(k)
+selection toggle: key(x)
+selection open: key(o)
+# legacy [verb noun] commands to be removed at a later time
```

Comment:
I have mixed feelings bout leaving these, because people may just continue using them and still be surprised when they are removed later. 

How do we want to handle these deprecations in general? Just remove them? Replace them with a notification? Leave them with a note like has been done here? Cc/ @rntz @knausj85 

Would be interesting to use the code that generates help to generate a diff of commands that have been added / removed between two commits so people can easily check what has changed when they merge